README.md

```diff
@@ -1,22 +1,21 @@
-This repo is for review of requests for signing shim.  To create a request for review:
+This repo is for review of requests for signing shim. To create a request for review:
 
-- clone this repo
+- clone this repo (preferably fork it)
 - edit the template below
 - add the shim.efi to be signed
 - add build logs
 - add any additional binaries/certificates/SHA256 hashes that may be needed
 - commit all of that
 - tag it with a tag of the form "myorg-shim-arch-YYYYMMDD"
-- push that to github
+- push it to GitHub
 - file an issue at https://github.com/rhboot/shim-review/issues with a link to your tag
 - approval is ready when the "accepted" label is added to your issue
 
 Note that we really only have experience with using GRUB2 or systemd-boot on Linux, so
 asking us to endorse anything else for signing is going to require some convincing on
 your part.
 
-Check the docs directory in this repo for guidance on submission and
-getting your shim signed.
+Check the docs directory in this repo for guidance on submission and getting your shim signed.
 
 Here's the template:
```

Comment:
Do we want to delete everything above this line?

Reply:
here is an example from another vendor: https://github.com/jsegitz/shim-review/blob/SUSE-liberty-15.8-20240514/README.md

We could, but it is not usually done

Reply:
Gotcha.